### PR TITLE
OCP/PCI-DSS: Add responses for Req-12

### PR DESCRIPTION
**Pull request by JAORMX**
Most of these are related to setting appropriate prodsec processes and
are not technical controls.

Signed-off-by: Juan Antonio Osorio Robles <jaosorior@redhat.com>

### CODE REVIEW

**Review by jhrozek — COMMENTED** [review]
Looks good, let me know if you want to merge as-is or include `audit_error_alert_exists`

File: controls/pcidss_ocp4.yml [hunk 1]
```diff
@@ -3151,17 +3228,27 @@ controls:
...
-    status: pending
+    notes: |-
+      This is a requirement for the incident response program, and its the responsibility of the
+      payment entity to include such information. However, the OpenShift Container Platform
+      does come with a monitoring stack that allows for transforming metrics into relevant
+      alerts [1]
+
+      [1] https://docs.openshift.com/container-platform/4.9/monitoring/managing-alerts.html
+    # TODO(jaosorior): is there a relevant rule we could write for this? It's enabled
+    #                  by default and it cannot be turned off.
```
[the remark this answers is not in the document]
Comment (JAORMX) [reply controls/pcidss_ocp4.yml:3239]: I think we should create a more general rule that checks that alertmanager is configured. You might have alerts in the system but nothing consuming them. I'll create a Jira card and point to this control.

**Comment by jhrozek** [comment]
On Thu, Nov 11, 2021 at 05:13:51AM -0800, Juan Osorio Robles wrote:
> @JAORMX commented on this pull request.
> 
> 
> 
> > @@ -3151,17 +3228,27 @@ controls:
>        monitoring systems.
>      levels:
>      - base
> -    notes: ''
> -    status: pending
> +    notes: |-
> +      This is a requirement for the incident response program, and its the responsibility of the
> +      payment entity to include such information. However, the OpenShift Container Platform
> +      does come with a monitoring stack that allows for transforming metrics into relevant
> +      alerts [1]
> +
> +      [1] https://docs.openshift.com/container-platform/4.9/monitoring/managing-alerts.html
> +    # TODO(jaosorior): is there a relevant rule we could write for this? It's enabled
> +    #                  by default and it cannot be turned off.
> 
> I think we should create a more general rule that checks that alertmanager is configured. You might have alerts in the system but nothing consuming them. I'll create a Jira card and point to this control.

Fine, I'll ack this PR in the meantime.

**Review by jhrozek — APPROVED** [review]
/lgtm

**Review by Vincent056 — APPROVED** [review]
/lgtm

**Pull request merged**